geode-management/src/main/java/org/apache/geode/management/internal/operation/RestoreRedundancyResponseImpl.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.internal.operation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.geode.management.api.ClusterManagementResult;
+import org.apache.geode.management.runtime.RegionRedundancyStatusSerializable;
+import org.apache.geode.management.runtime.RestoreRedundancyResponse;
+
+public class RestoreRedundancyResponseImpl
```

Comment:
let's just make RestoreReduncyResponse a concrete class. I really don't see much benefit in having an interface.

Reply:
also you don't have to call it Serializable in the class name, it fact that it implements OperationResult is already Serializable

Reply:
I used serializable in the name because the class is basically a copy of an existing one with the serializability being a distinguishing feature. This is one of those points of question where, should I just modify the RegionRedundancyStatus class to make it serializable?

Reply:
> let's just make RestoreReduncyResponse a concrete class. I really don't see much benefit in having an interface.

I agree. It adds a layer of isolation that is not necessary.

Reply:
> I used serializable in the name because the class is basically a copy of an existing one with the serializability being a distinguishing feature. This is one of those points of question where, should I just modify the RegionRedundancyStatus class to make it serializable?

that one is already DataSerializable, so you shouldn't have to make it Serializable. I think you can just use whatever they had used as the result before, just move them to the management module. If the object are used in locator/server function calls, then they must have been Serializable/DataSerializable already.